locopy/redshift.py

```diff
@@ -289,7 +292,14 @@ def load_and_copy(
         # generate the actual splitting of the files
         # We need to check if IGNOREHEADER is set as this can cause issues.
         ignore_header = get_ignoreheader_number(copy_options)
-        upload_list = split_file(local_file, local_file, splits=splits, ignore_header=ignore_header)
+
+        if os.path.isdir(local_file):
+            upload_list = [local_file + "/" + x for x in os.listdir(local_file)]
```

Comment:
Might make sense to use `PathLib` or something else instead of `/` to ensure it is platform agnostic?